pasha/__init__.py

```diff
@@ -0,0 +1,105 @@
+
+# Distributed under the terms of the BSD 3-Clause License.
+# The full license is in the file LICENSE, distributed with this software.
+#
+# Author: Philipp Schmidt <philipp.schmidt@xfel.eu>
+# Copyright (c) 2020, European X-Ray Free-Electron Laser Facility GmbH.
+# All rights reserved.
+
+__version__ = '1.0.0'
+
+
+from .context import SerialContext, ThreadContext, ProcessContext  # noqa
+from .functor import (  # noqa
+    SequenceFunctor, NdarrayFunctor, DataArrayFunctor, ExtraDataFunctor)
+
+
+_default_context = ProcessContext()
```

Comment:
This means we're doing a bit of work on import, which is usually best avoided if practical. Digging down the layers, I think this will create a pipe and several Posix semaphores. I think it would be fairly easy to defer this by constructing it in `get_default_context()` the first time it's called.